app/views/interstitial/ViewNewsAndUpdatesView.scala.html

```diff
@@ -17,38 +17,31 @@
 @import config.ConfigDecorator
 @import controllers.auth.requests.UserRequest
 @import uk.gov.hmrc.renderer.TemplateRenderer
-@import uk.gov.hmrc.play.bootstrap.binders.SafeRedirectUrl
+@import views.html.components.{H1, H2, Link, P}
+
 @import scala.concurrent.ExecutionContext
-@import views.html.components.{H1, Link, P, H2}
 
 @this(
-    mainView: newMain,
-    h1: H1,
-    h2: H2,
-    p: P,
-    link: Link
+        mainView: newMain,
+        h1: H1,
+        h2: H2,
+        p: P,
+        link: Link
 )
 
-@(redirectUrl: String)(implicit request: UserRequest[_], configDecorator: ConfigDecorator, templateRenderer: TemplateRenderer, messages: play.api.i18n.Messages, ec: ExecutionContext)
-@mainView(
-    pageTitle = messages("label.news_and_updates"),
-    showBackLink = true
-){
-
-    @h1("label.news_and_updates", elmId =Some("newsHeading"))
-
-    <section class="app-markdown">
-        @h2("label.percentage_points_uplift_in_NIC", elmId=Some("nic"))
-        @p(Text(messages("label.national_insurance_contributions_will_increase_by_1.25_percentage_points")))
-        @p(Text(messages("label.the_increase_will_apply_to")))
-
-        <ul class="govuk-list govuk-list--bullet">
-            <li>@messages("label.class_1_paid_by_employees")</li>
-            <li>@messages("label.class_4_paid_by_self_employed")</li>
-            <li>@messages("label.secondary_class_1_1a_1b_paid_by_employers")</li>
-        </ul>
-        @p(Text(messages("label.the_increase_will_not_apply_if_you_are_over_the_state_pension_age")))
-        @link(link = messages("healthAndSocialCareLevy.link.url"), messageKey = "label.prepare_for_the_health_and_social_care_levy", attrTarget = true)
-    </section>
+@(redirectUrl: String, newsAndContentModelList: List[NewsAndContentModel])(implicit request: UserRequest[_], configDecorator: ConfigDecorator, templateRenderer: TemplateRenderer, messages: play.api.i18n.Messages, ec: ExecutionContext)
+    @mainView(
+        pageTitle = messages("label.news_and_updates"),
+        showBackLink = true
+    ) {
 
-}
+        @h1("label.news_and_updates", elmId = Some("newsHeading"))
+        @for(newsAndContentModel <- newsAndContentModelList) {
+            <section class="app-markdown">
+                <h2 id="@newsAndContentModel.newsSectionName" autofocus>@newsAndContentModel.shortDescription</h2>
```

Comment:
This won't work. Only the selected news should have the autofocus attribute.

Reply:
Done